client/src/components/ui/itemrow/SendRow.svelte

```diff
@@ -13,8 +17,25 @@
     export let category: Category['name'];
     export let title: Document['title'];
     export let target: Snapshot['target'];
+    export let creation: Snapshot['creation'];
+
+    function redirectHandler() {
+        window.location.href = `/track?id=${doc}`;
+    }
+
+    const targetName = target ? $allOffices[target] : '';
```

Comment:
I prefer to be more explicit with conditionals. Instead of checking `if (target)`, it is more readable to say `if (target === null)`. The intent is much clearer.
```suggestion
    $: office = target === null ? $allOffices[target] : null;
    $: targetName = office ?? '[No Office]';
```